### PR TITLE
Add gcc version in docs/BUILD-conan.md

### PR DESCRIPTION
**Pull request by segfaultxavi**
It's specified like this in BUILDLIN.md but it's missing in BUILD-conan.md

### CODE REVIEW

File: docs/BUILD-conan.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ NOTE: Commands are using `\` as marker for line continuations
 
 ## Prerequisites
 
+ * gcc 9.2
```
Comment (Jaguar0625) [inline docs/BUILD-conan.md:9]: please include recommended versions for all supported compilers (gcc, clang, msvc)

Comment (Jaguar0625) [reply docs/BUILD-conan.md:9]: conan supports all three

[ofc, only one is needed]

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
Please make it clearer that conan works with multiple supported compilers, not just gcc.

**Review by Jaguar0625 — APPROVED** [review]

**Review by gimer — APPROVED** [review]

**Comment by Jaguar0625** [comment]
Merged as 0125a1065.

**Pull request closed without merging**